BUG-TRIAGE.md

```diff
@@ -0,0 +1,32 @@
+# Jaeger Bug Triage
+
+This procedure describes the steps project maintainers and approvers should take to triage a bug report.
```

Comment:
for reference https://github.com/open-telemetry/opentelemetry-specification/issues/3821

Reply:
And the actual process doc: https://github.com/open-telemetry/opentelemetry-specification/pull/3990/files

Reply:
To clarify, I don't think we want as complex workflow as OTEL Spec, but I do like to have a state diagram that defines when & how issues are transitioned between states, with the aim that there is no state in which an issue can sit infinitely (since it just creates a never-clearing backlog)